topic: "scan"
angular_step: 5
sampling_freq: 50
max_angle_deg: 90
min_angle_deg: -90
min_range: 0.1
max_range: 10

